### PR TITLE
fix/feat: major changes

### PR DESCRIPTION
**Pull request by i-dkz**
- completely restarted the project (i saved a backup of the old project don't worry)
- switched to tailwind/shadcnUI
- project is now connected to supabase

### CODE REVIEW

File: .DS_Store
Comment (jonbiemond) [inline .DS_Store:1]: Can you take `.DS_Store` out of the commit?
You can ignore it with a `.gitignore` in the top level directory. Containing:
```
*
.DS_Store
```

**Review by jonbiemond — CHANGES_REQUESTED** [review]
Looks awesome, excited about the new UI framework and having Supabase set up!
One minor quibble, shouldn't `/ewp` be the top level directory? Like, the folder itself shouldn't exist in the repo, that folder should be the repo. If it's a not as simple as just copying the contents one level higher don't bother.

File: .DS_Store
Comment (jonbiemond) [reply .DS_Store:1]: Note that removing `/ewp` and then refactoring all the contents up a level also resolves this without having to create another `.gitignore` file.

Comment (i-dkz) [reply .DS_Store:1]: image

ewp is the top level, and there is a .gitignore file with .DS_store in it. I can't delete the .DS_store it's a mac thing.

Comment (jonbiemond) [reply .DS_Store:1]: I think you should be able to excluded `.DS_Store` from the repo now without deleting it.
`git rm --cached .DS_Store`

Comment (i-dkz) [reply .DS_Store:1]: I don't even see the ds_store file

Comment (jonbiemond) [reply .DS_Store:1]: I see it in [Files changed](https://github.com/Alschematix/StudyProject/pull/16/files)

Comment (jonbiemond) [reply .DS_Store:1]: I dropped it with https://github.com/Alschematix/StudyProject/pull/16/commits/6166fe56532410aad73a5b4b120e77acb5d98750

**Review by jonbiemond — APPROVED** [review]

**Pull request merged**